### ISSUE TITLE
✨️ Option to switch off or customize secondary side item

### ISSUE THREAD

**Issue by CyberRooke** [jurialmunkey/skin.arctic.fuse#712]
### Skin section

Widgets

### Feature description

Could we have the option to either switch off or be able to customize the secondary side item as we can do with the primary. 

### Why is this feature needed? Please justify and explain.

Personally I would rather be able to switch it off as I prefer just 1 item there, but I can see how 2 creates consistency with the other hubs and libraries... could we then get the option to customize it or choose other options for it at the very least as currently it is just the shutdown menu and there are multiple other ways to access the shutdown menu... It'll then seem like a waste of what could be a useful item for users who use any other means to access the shutdown menu 

### Checklist

- [X] I have made a separate issue for each feature I am requesting.
- [X] I have searched the issues and this feature has not been requested.
- [X] I am using the most recent skin version.

**Comment by L0bsang** [comment]
> as currently it is just the shutdown menu

For me when menu is minimized the second side item opens the side-menu and is quite handy for quick hub navigation. 
Does it open the shutdown/options menu for you?

**Comment by CyberRooke** [comment]
> > as currently it is just the shutdown menu
> 
> For me when menu is minimized the second side item opens the side-menu and is quite handy for quick hub navigation. 
> Does it open the shutdown/options menu for you?

Yeah, it opens the shutdown menu for me... there was previously an option to switch between the shutdown menu and the sidemenu dialog, but it's no longer available... are you on topbar?

**Comment by L0bsang** [comment]
No, I'm using sidebar always minimised. I guess this issue only applies to topbar.

Extra comments:
Tried topbar for a good while, it didn't make sense to me as a navigation option, experimental for good reason.
- Not only cannot I have labels on it (more family friendly to have labels), the sidebar is also still always present and the navigation in complex mode is horizontal (meaning categories function identically to a topbar). 
- So two menus instead of one and the other is redundant if category visibility is set to always visible for all categories.So now I just use the sidemenu to jump around between hubs (at least in complex mode, in classic mode I have a different setup where topbar might make sense.

**Comment by CyberRooke** [comment]
I see... I've seen someone else say Topbar navigation doesn't make sense.. I struggle to understand how... The sidemenu is the one that's hard for me, especially with linked categories and spotlight... the navigation always passes through the spotlight.... doesn't seem fluid, whereas with Topbar, the side item will allow you to skip the spotlight when switching between linked categories,  creating a kind circular motion.

Not seeing labels is a good point, though... but I only use hubs in my personal setup, and I can understand all the icons, and they are way more minimal... In my living room setup, I only have one page.. just the home page, then I use different categories and widgets, so there will be no need for switching through hubs, which is family friendly
